src/style/utils/at_opacity.js

```diff
@@ -0,0 +1,9 @@
+export default (base) => {
+  const parse = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(base);
+
+  const r = parseInt(parse[1], 16),
+      g = parseInt(parse[2], 16),
+      b = parseInt(parse[3], 16);
+
+  return opactiy => `rgba(${r},${g},${b},${opactiy})`;
```

Comment:
if we use map, then we can just write ```rgba(${rgb.join(',')},${opactiy})```

Reply:
typo on `opactiy`

Reply:
documentation on this would also be good please